.storybook/stories/paragraphs/Gallery.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {ComponentProps} from "react";
+import GalleryParagraph from '@components/paragraphs/stanford-gallery/gallery-paragraph';
+import { getStoryBookGalleryImage } from '../storybook-entities';
+
+type ComponentStoryProps = ComponentProps<typeof GalleryParagraph> & {
+
+}
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta: Meta<ComponentStoryProps> = {
+  title: 'Design/Paragraphs/Gallery',
+  component: GalleryParagraph,
+  tags: ['autodocs'],
+  argTypes: {
+    numberofimages: {
+      control: {
+        type: 'number',
+        min: 1,
+        max: 20,
+      }
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<ComponentStoryProps>;
+
+// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+export const Gallery: Story = {
+  render: ({numberofimages, ...args}) => {
+    return <GalleryParagraph { ...args} />
+  },
+  args: {
+    paragraph:{
+      __typename: 'ParagraphStanfordGallery',
+      composition: {},
+      langcode: {},
+      created: {
+        offset: "",
+        timestamp: Math.round(new Date().getTime() / 1000),
+        time: new Date().toISOString(),
+        timezone: "America/Los_Angeles"
+      },
+      id: "1e0c21ee-7cb8-4eff-8877-862d16c9885c",
+      suGalleryButton: {
+        url: "http://stanford.edu",
+        title: "Button text",
+        internal:false,
+      },
+      suGalleryDescription: {
+        processed: "<p>Pepper jack fromage frais pecorino cheesecake cheesy grin camembert de normandie macaroni cheese the big cheese.</p>"
+      },
+      suGalleryHeadline: "This is a Gallery Headline",
+      numberofimages: 4,
+      suGalleryImages: [ 
+        getStoryBookGalleryImage(),
```

Comment:
I can't figure out how to do a for loop for however many times the image is wanted from the control.  I was using this video for guidance. https://www.youtube.com/watch?v=FUKpWgRyPlU

Reply:
@jenbreese you can see my last commit to this branch https://github.com/SU-SWS/cardinalsites-nextjs/pull/3/commits/2948d969735fdcf44c095e62db4d7dd2978d698b

Reply:
@pookmish Thank you. That's exactly what I wanted it to do. I'm going to study the change a bit so I can understand it.